src/systems/filecoin_blockchain/message_pool/message_syncer.md

```diff
@@ -11,7 +11,7 @@ TODO:
 
 # Message Propagation
 
-Messages are propagated over the libp2p pubsub channel `/fil/messages`. On this channel, every [serialised `SignedMessage`](data-structures.md#messages) is announced.
+Messages are propagated over the libp2p pubsub channel `/fil/messages`. On this channel, every serialised `SignedMessage` is announced (see {{<sref message>}}).
```

Comment:
`message_syncer` is a better reference

Reply:
but this is the message_syncer file already so useless to self-link. leaving as is